lib/routes/sciencedirect/call-for-paper.ts

```diff
@@ -0,0 +1,111 @@
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/call-for-paper/:subject',
+    categories: ['journal'],
+    example: '/sciencedirect/call-for-paper/education',
+    parameters: {},
```

Comment:
Missing path parameter description
 https://github.com/DIYgod/RSSHub/blob/22661807c067325cf08afd7a6478146872e0db6e/lib/routes/sciencedirect/call-for-paper.ts#L5